spec/interjectable_spec.rb

```diff
@@ -52,6 +75,16 @@
           expect(instance.some_other_class).to eq(:fake_other_class)
         end
       end
+
+      context "with a subclass" do
+        let(:subclass) { Class.new(klass) }
+        let(:subclass_instance) { subclass.new }
+
+        it "does not error if the method exists on the super klass" do
```

Comment:
nit: `superclass` not `super klass` right?

Reply:
lol, yes.